RobotApplication.java

```diff
@@ -1 +1,33 @@
+package com.company;
 
+public class RobotApplication {
+
+    public static void main(String[] args) {
+        RobotConnectionManager robotConnection = null;
+        moveRobot(robotConnection, 4, 5);
+    }
+
+    public static void moveRobot(RobotConnectionManager robotConnectionManager, int toX, int toY) {
+        boolean check = false;
+        RobotConnection robotConnection = null;
+        for (int i = 1; !check && i <= 3; i++) {
+            try {
+                robotConnection = robotConnectionManager.getConnection();
+                robotConnection.moveRobotTo(toX, toY);
+                check = true;
+            } catch (RobotConnectionException ex) {
+                if (!check && i == 3) {
```

Comment:
remove "!check".
it is unnecessary here

Reply:
ok